### PR TITLE
Added Apertass

### PR DESCRIPTION
**Pull request by Pensionato**

### CODE REVIEW

**Comment by Pensionato** [comment]
> Can you provide a horizontal image 600x400px?

Sure

File: src/recipes/apertass.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
         "Pour a small bottle of Tassoni Soda cedar and add a tip of Aperol, completing with the Soda.",
         "Stir gently. Garnish with a slice of orange."
     ],
-    "image": "apertass.jpg>",
+    "image": "apertass600x400.jpg>",
```
Comment (alfg) [inline src/recipes/apertass.json]: This should still be `apertass.jpg` and the image filename should also be `apertass.jpg`

Comment (Pensionato) [reply src/recipes/apertass.json]: Done

File: src/recipes/apertass.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+            "measure": "slice",
+            "ingredient": "Orange"
+        }
+    ],
+    "directions": [
+        "Cool a tumbler with ice, drain the water formed with a strainer.",
+        "Pour a small bottle of Tassoni Soda cedar and add a tip of Aperol, completing with the Soda.",
+        "Stir gently. Garnish with a slice of orange."
+    ],
+    "image": "apertass.jpg>",
```
Comment (alfg) [inline src/recipes/apertass.json]:
```suggestion
    "image": "apertass.jpg",
```

**Comment by alfg** [comment]
Thanks for the fixes!

**Pull request merged**